services/QuillLMS/app/queries/admin_diagnostic_reports/diagnostic_aggregate_view_query.rb

```diff
@@ -26,6 +29,7 @@ def initialize(timeframe_start:, timeframe_end:, school_ids:, aggregation:, grad
     end
 
     def materialized_views = [filter_view, performance_view]
+    def fallback_views = materialized_views
```

Comment:
This method feels like a passthrough.  Maybe remove?

Reply:
This is probably harder to see without the upcoming code, but `fallback_views` is used by `ViewQueryFallback`. I'm decoupling that code from `materialized_views` so that there can be multiple types , e.g. the upcoming `table_from_query_views`. A class can assign any combination of `ConfigurableViews` here.